pkg/skaffold/watch/triggers.go

```diff
@@ -109,3 +114,51 @@ func (t *manualTrigger) Start() (<-chan bool, func()) {
 
 	return trigger, func() {}
 }
+
+// notifyTrigger watches for changes when fsnotify
+type fsNotifyTrigger struct {
+}
+
+// Debounce tells the watcher to not debounce rapid sequence of changes.
+func (t *fsNotifyTrigger) Debounce() bool {
+	return false
```

Comment:
I think we should activate debounce, here.
If a lot of changes are made to the sources, very quickly, we don't want to trigger a rebuild too many times.

Reply:
Oh, I see why you returned `false`, it doesn't work with `true`...